examples/webhook-signing/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "types": ["../../types/2019-12-03"],
```

Comment:
This shouldn't be needed once it lands, but I guess we can keep it in till then?

Reply:
Yeah I wasn't sure if this is necessary or not, happy to remove once v8 lands.